### PR TITLE
Model submission for RMF Sensor Configuration 1 by CERBERUS

### PR DESCRIPTION
**Pull request by mihirk284**
Model Submission for the CERBERUS RMF(Resilient Micro Flyer) with Sensor Configuration 1.

image

### CODE REVIEW

File: submitted_models/cerberus_rmf_sensor_config_1/specifications.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+<!-- This is a Markdown description of a robot model submitted for inclusion in the
+DARPA Subterranean Challenge Technology Repository -->
+
+# CERBERUS-Aerial Scout
+This specifications.md file is a description and proof of virtual model validation for
+the CERBERUS Aerial Scout with Sensor Configuration 1. This robot may be launched using a launch command with the variable name `CERBERUS_RMF_SENSOR_CONFIG_1`.
+
+## Description
+CERBERUS RMF (Resilient Micro Flyer) Aerial Scout is a highly autonomous system capable of being deployed in an unknown environment and proceeding for its mapping up to the point of automatic return to home. The platform basis is designed by Flyability but the rest is custom integration and new algorithms.
+In this configuration, CERBERUS Aerial Scout is equipped with the following sensors:
```
Comment (angelacmaio) [inline submitted_models/cerberus_rmf_sensor_config_1/specifications.md:10]: Lines 10-11 refer to a different robot model

**Comment by acschang** [comment]
The initial assessment of the CERBERUS RMF model is complete. We will need to add optical frame publishers for the cameras. I will merge this to a new branch `submitted_models/cerberus_rmf_sensor_config_1` and create a [follow-up PR](https://github.com/osrf/subt/pull/722) targeting master to continue the review process.

**Comment by mihirk284** [comment]
Thank you for iterating on the PR @acschang @angelacmaio  
Can you please let me know what changes need to be made on our end for moving forward with the model review process?

**Comment by acschang** [comment]
> Thank you for iterating on the PR @acschang @angelacmaio
> Can you please let me know what changes need to be made on our end for moving forward with the model review process?

No further action is required at this time.

**Pull request closed without merging**